lib/ABI/FunctionType/Conversion.cpp

```diff
@@ -282,6 +299,10 @@ static bool verifyAlignment(const abi::Definition &ABI,
   if (*Offset == NextOffset) {
     revng_log(Log, "Argument slots in perfectly.");
     return true;
+  } else if (NextAlignment == 0) {
+    // This is the last argument.
+    // TODO: is there anything more we can check in here?
+    return true;
```

Comment:
What's special about the last argument?

Reply:
The padding after it seems to be somewhat inconsistent in some cases.

I still have some doubts about this (hence the TODO), but TL;DR of this change is: if we know this is the last argument, we don't really care about anything (as in, padding... or lack thereof) that comes after it.

Reply:
OK, elaborate the comment a bit.

Reply:
Done.

```cpp
    // This is the last argument.
    //
    // Since we often find trailing padding to be inconsistent, it would be
    // hard to enforce it, as such, just assume that if we know this argument
    // is the last one, it's always right.
```